tests/testthat/test-bind_marginals.R

```diff
@@ -0,0 +1,9 @@
+test_that("bind_marginals works", {
+  expect_equal(length(bind_marginals(cop_ig(theta = 1, alpha=2), rnorm(50, 2), rnorm(50, 4))), 3)
+  expect_equal(names(bind_marginals(cop_ig(theta = 1, alpha=2), rnorm(50, 2), rnorm(50, 4))[1]), "copula")
```

Comment:
Can combine the three `expect_equal` calls that check the names; names should equal the vector `c("copula", "dst_x", "dst_y")`.